src/BenchmarksApps/TodosApi/TodosApi.openapi.json

```diff
@@ -8,7 +8,7 @@
     "/throw": {
       "get": {
         "tags": [
-          "TodosApi"
+          "TodosApi, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null"
```

Comment:
This seems not ideal

Reply:
Oh interesting. I'm assuming you mean the document name seemingly being the full assembly name? This was produced by our own package (MS.Ext.ApiDescription.Server) but not sure why it changed. It's possible the previous version was produced by the Swashbuckle CLI and there's a subtle difference somewhere.

Reply:
Wait, is this from an old commit? The latest doesn't have this, perhaps it was the Swashbuckle CLI that added it?

Reply:
https://github.com/domaindrivendev/Swashbuckle.AspNetCore/blob/master/src/Swashbuckle.AspNetCore.Cli/HostingApplication.cs#L68-L72